day_two_guide/topics/proc_scaling-etcd-ansible.adoc

```diff
@@ -36,6 +36,11 @@ master-2.example.com
 etcd0.example.com <1>
 ----
 <1> Add these lines.
++
+[NOTE]
+====
+Replace the old etcd host entry with the new etcd host entry in the inventory file. While replacing the older etcd host, you must create a copy of `/etc/etcd/ca/` directory. Alternatively, you can redeploy etcd ca and certs before scaling up the etcd hosts.
```

Comment:
Should etcd host be `etcd host`?

Reply:
umm, I checked for similar entries in the docs. For eg etcd member, etcd backup etc. It's written in plain format. So I think we should keep etcd host in the same manner? Please correct me if I'm wrong.

Reply:
Right. But in this case etcd host appears to be a value in the Ansible inventory file referenced directly above your note.

Reply:
You are correct. Thanks for the clarification. I addressed the change.